src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC, SetStateAction } from 'react';
+import classNames from 'classnames';
+import { FilterTodosBy } from '../../types/FilterTodosBy';
+
+type Props = {
+  setFilterBy: (string: SetStateAction<FilterTodosBy>) => void,
```

Comment:
The type of the argument 'string' in setFilterBy function should be FilterTodosBy instead of SetStateAction<FilterTodosBy>.